server/pulp/server/managers/content/query.py

```diff
@@ -113,28 +113,29 @@ def get_content_unit_by_keys_dict(self, content_type, unit_keys_dict, model_fiel
         """
         Look up an individual content unit in the corresponding content type
         collection using the given keys dictionary.
-        @param content_type: unique id of content collection
-        @type content_type: str
-        @param unit_keys_dict: dictionary of key, value pairs that can uniquely
+        :param content_type: unique id of content collection
+        :type content_type: str
+        :param unit_keys_dict: dictionary of key, value pairs that can uniquely
                                identify a content unit
-        @type unit_keys_dict: dict
-        @param model_fields: fields of each content unit to report,
+        :type unit_keys_dict: dict
+        :param model_fields: fields of each content unit to report,
                              None means all fields
-        @type model_fields: None or list of str's
-        @return: content unit from the content type collection that matches the
+        :type model_fields: None or list of str's
+        :return: content unit from the content type collection that matches the
                  keys dict
-        @rtype: dict
-        @raise: ValueError if the unit_keys_dict is invalid
-        @raise: L{MissingResource} if no content unit in the content type
+        :rtype: dict
+        :raise: ValueError if the unit_keys_dict is invalid
```

Comment:
should be :raises ValueError:

http://sphinx-doc.org/domains.html#info-field-lists